storage/src/adb/any.rs

```diff
@@ -382,12 +382,14 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Array, H: CHasher> Any<E, K, V,
         let end_loc = start_loc + ops.len() as u64 - 1;
         let end_pos = leaf_num_to_pos(end_loc);
 
+        // We could avoid the temp hasher if we were to pre-compute all digests and put them in a
+        // temp vector.
+        let mut temp_hasher = H::new();
```

Comment:
Let's do the precompute and avoid the temp_hasher. I think the cleanliness is worth whatever small performance issue (if any) it may lead to.

Reply:
done